device/device.go

```diff
@@ -322,6 +325,19 @@ func NewDevice(tunDevice tun.Device, bind conn.Bind, logger *Logger) *Device {
 	return device
 }
 
+// BatchSize returns the BatchSize for the device as a whole which is the max of
+// the bind batch size and the tun batch size. The batch size reported by device
+// is the size used to construct memory pools, and is the allowed batch size for
+// the lifetime of the device.
+func (device *Device) BatchSize() int {
+	sz := device.net.bind.BatchSize()
+	dSz := device.tun.device.BatchSize()
+	if sz < dSz {
+		sz = dSz
+	}
+	return sz
```

Comment:
`size` is used over `sz` everywhere else.

(Side note: still no generic min/max functions in std? Humph.)

Reply:
Updated in https://github.com/WireGuard/wireguard-go/pull/64/commits/cd503f9a5e9a9870c1e70ea7ea73930d6c0631a0.